conda-recipes/python-microscopy/menu-windows.json

```diff
@@ -15,7 +15,7 @@
                 "script": "${PREFIX}/Scripts/dh5view.exe",
                 "scriptarguments": [],
 
-                "name": "dh5view",
+                "name": "PYMEImage",
```

Comment:
could we make this have the old name in brackets for a bit? At least until people have gone through one install / update cycle and our documentation is fixed.

Reply:
Ditto for VisGUI below. Don't worry about the gnome stuff - it must be 5 years or so since anyone used that and if you are on linux you should be able to figure it out.

Reply:
Done.